### PR TITLE
Add 6.0.2 release notes

### PR DESCRIPTION
**Pull request by kartheekp-ms**

### CODE REVIEW

**Review by erdembayar — APPROVED** [review]

File: docs/release-notes/NuGet-6.0.md [hunk 1]
```diff
@@ -15,11 +15,15 @@ NuGet distribution vehicles:
 |:---|:---|:---|
 | [**6.0.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.0.0](https://visualstudio.microsoft.com/downloads/) | [6.0.0](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
 | **6.0.1** | [Visual Studio 2022 version 17.0.2](https://visualstudio.microsoft.com/downloads/) | N/A |
+| [**6.0.2**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.0.11](https://visualstudio.microsoft.com/downloads/) | [6.0.6](https://dotnet.microsoft.com/download/dotnet-core/6.0)<sup>1</sup> |
```
Comment (erdembayar) [inline docs/release-notes/NuGet-6.0.md:18]: Actually 6.0.6 is runtime version. Actual  sdk versions are `SDK 6.0.106`, `SDK 6.0.301`. Which one usually we include here?

Comment (zivkan) [reply docs/release-notes/NuGet-6.0.md:18]: NuGet ships in the .NET SDK, not the runtime, so it doesn't make sense for us to use the runtime version (in this row, or the above rows)

Comment (kartheekp-ms) [reply docs/release-notes/NuGet-6.0.md]: My bad I intended to add `6.0.301` version but copied runtime version  by mistake. Fixed in the latest commit.

Comment (erdembayar) [reply docs/release-notes/NuGet-6.0.md]: Hmmm. Here pushing new commit didn't reset my approval. Should we reset any approvals here too?

Comment (kartheekp-ms) [reply docs/release-notes/NuGet-6.0.md]: Sure, we can track that separately. Are there any comments that I should address in this PR, please let me know?

Comment (erdembayar) [reply docs/release-notes/NuGet-6.0.md]: I don't have any other comment here.

**Pull request merged**